_ecosys/consumers/Telerik-RadGrid-for-ASP.NET-Ajax.md

```diff
@@ -2,6 +2,6 @@
 layout: default
 category: consumers
 title: "Telerik RadGrid for ASP.NET Ajax"
-link: "http://www.telerik.com/products/odata-support.aspx"
+link: "https://www.telerik.com/products/odata-support.aspx"
```

Comment:
It looks like this has been discontinued and we should probably delete this entry.